### PR TITLE
Add 1st endpoint for posts + redirect | Add get request to homepage

### PR DESCRIPTION
**Pull request by DevisWeb**
**This Pull contains:**
- Endpoint to get all posts on mainpage (index.js)
- In posts routers: just redirect from http://localhost:3000/posts to '../'

Further:  comment-list for endpoints that need to be added
  
___

For more details see corresponding commit

### CODE REVIEW

**Comment by DevisWeb** [comment]
@vibueno (referring to our talk yesterday)

Depending on how we finally want to set up the project, this version fits, if we want to have all posts directly displayed on the homepage.

Since this still can be easily changed at any time of this project status and the requested task  ("sending a basic string") for our first endpoint works in any case now, I would suggest that we just merge this version into 'dev'. so that we can proceed from this point :-) and split up the new tasks on Monday as soon as the remaining endpoints are added and the middlewares are separated in the controller (that can be done until or also quickly together during group session), as so far we only have to copy/paste and test if each endpoint is correct by sending a basic message. 

Remaining tasks besides reading the documentations we got:
grafik

In the second card on trello (for separating the middlewares), I've listed the endoints we've defined so far on Friday, of which the middlewares yet have to be added so that we can easily jump in on Monday.

**Comment by vibueno** [comment]
I have had a look and it does what we want, but I find it clearer if we do it the other way around: redirecting on the main page -> /posts. See #4

What do you think?

File: controllers/posts.js [hunk 1]
```diff
@@ -1,19 +1,19 @@
 const postsController = {
-    
-  };
-  
-module.exports = postsController;
+  logAllPostsRedirect:
+    ('../posts',
```
Comment (monoamro) [inline controllers/posts.js:3]: Do we have to have the '../posts' path here? shouldn't be in the routes file?

File: index.js [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
+// import routes
 const postsRoutes = require('./routes/posts');
 
-app.use("/posts", postsRoutes);
\ No newline at end of file
+// use postsRoutes
+app.use('/posts', postsRoutes);
+
+// get request to homepage
+app.get('/', (req, res) => res.send('main page - will show all posts'));
```
Comment (monoamro) [inline index.js:12]: Is there a reason why this is also here? I think it's enough to have in the router file.

**Review by monoamro — COMMENTED** [review]
i just left some comments here

File: index.js (hunk 2 above)
Comment (DevisWeb) [reply index.js:12]: this is the root endpoint = the homepage ;-)

File: controllers/posts.js (hunk 1 above)
Comment (DevisWeb) [reply controllers/posts.js:3]: this is about handling a redirect, that's why ;-)

**Pull request closed without merging**